tests/mesolve/test_mesolve.py

```diff
@@ -33,7 +33,7 @@ def test_mesolve_batching():
     b_rho0 = len(rho0_batched)
     num_t_save = 51
     t_save = np.linspace(0.0, delta / (2 * np.pi), num_t_save)  # a full tour
-    solver = tq.solver.Rouchon(dt=1e-4, order=1)
+    solver = tq.mesolver.Rouchon1(dt=1e-4)
```

Comment:
I don't like that we should call `tq.mesolver` instead of `tq.solver`. The solver options should be common to all functions, and then within the actual function it is dispatched to the appropriate QSolver. It's much more simple on the user-end.

Reply:
Agreed 👍 The correct solution is to have a common namespace for solver options coming from different places, from what I know this can only be done by creating a new file importing all the guys from the different places, and then exposing the file namespace -> see the proposed changes.

Reply:
It was also the chance to rename `solver` to the more accurate `solver_options`, while keeping the simple `tq.solver` API.